cmd/webhook.go

```diff
@@ -61,8 +61,8 @@ Supported registries:
 				return err
 			}
 
-			if cfg.MaxConcurrency < 1 {
-				return fmt.Errorf("--max-concurrency must be greater than 1")
+			if cfg.MaxConcurrentApps < 1 {
+				return fmt.Errorf("--max-concurrent-apps must be greater than 1")
```

Comment:
Can we enforce the value range in controllerCmd.Flags().IntVar(...)?

Reply:
I meant to say something like Flags().IntVar(..., min, max, ...), with min and max as parameters to IntVar func to use the built-in checking in IntVar func.

Reply:
Ah, I think I got it. I can replace this with
```
controllerCmd.Flags().IntVar(&cfg.MaxConcurrentApps, "max-concurrency", env.ParseNumFromEnv("MAX_CONCURRENT_APPS", 10, 1, 100), "maximum number of update threads to run concurrently")
```
so using `env.ParseNumFromEnv` but in this case we have to use env vars for MaxConcurrentApps and MaxConcurrentReconciles